CONTRIBUTING.rst

```diff
@@ -27,6 +27,21 @@ any inconsistency or opportunity for improvement, you can edit those too.
 Besides, the `wiki`_ is open for everybody to edit, so feel free to add
 new content.
 
+To build the docs, you must first create a development environment (see
+below) and then in the ``docs/`` directory run::
+
+    $ cd docs
+    $ make html
+
+After this, the new docs will be inside ``build/html``. You can open
+them by running an HTTP server:
+
+    $ cd build/html
+    $ python -m http.server
+    Serving HTTP on 0.0.0.0 port 8000 ...
```

Comment:
It doesn't render really well: 
![imagen](https://user-images.githubusercontent.com/18404378/29772633-f283c8ac-8bf9-11e7-82bc-05c8bdf9219f.png)

Maybe adding `$ Out: Serving HTTP on 0.0.0.0 port 8000 ...` would solve the problem, although it is not how linux shells show output.

Reply:
Only one colon missing, should have been `run::` :) Fixing!